# Haplotype Comparison and Analysis Tool (HCAT)
## Graphical User Interface (GUI)

### Overview
HCAT is a tool to help all scientists out there to analyze their haplotype sequence data faster and more reliable, by utilizing the strengths of a computer to automate repetitive tasks in a uniform way. No longer do you have to waste hours painstakingly categorizing sequence data into haplotypes, always fearing that you might slip up somewhere and by thus invalidating all the hard work you did.

With this tool, you can...
* Easily group sequences into haplotypes
* Compare those haplotypes to a reference sequence and show the variable sites and differences
* Automatically create haplotype tables and distance matrices and save them in a format readable by commonly used spreadsheet programs
* Cut all sequences freely and save them
* Translate sequences into codons and save them, all translation tables pre-configured

It can read in and write out sequences in different formats, like...
* Fasta
* Phylip (normal and for TCS (Clement et al., 2000))
* CSV (Comma Separated Value)

### Screenshots
Overview of all sequences, grouped together into haplotypes.

![Overview](https://user-images.githubusercontent.com/40519618/122640507-3daec080-d100-11eb-8cae-5a2328883751.png)


Showing all positions where the haplotypes differ from a master sequence.

![HaplotypeTable](https://user-images.githubusercontent.com/40519618/122640521-4b644600-d100-11eb-9161-2b4a355aefa8.png)


A dialog to translate the sequences into codons.

![SequnceTranslation](https://user-images.githubusercontent.com/40519618/122640760-7c914600-d101-11eb-9d38-a5c385d3f463.png)
